File: Atrasti.ReactNative/Atrasti/src/Api/Models/Responds/ProfilePage_Res.ts

import { User_Res } from './User_Res';
import { CompanyPage_Res } from './CompanyPage_Res';
import { AgentPage_Res } from './AgentPage_Res';
import { UserType_Mod } from './UserType_Mod';

export interface ProfilePage_Res {
    setup: boolean;
    isProfileOwner: boolean;
    user: User_Res;
    companyPage: CompanyPage_Res;
    agentPage: AgentPage_Res;
    userType: UserType_Mod;
}

File: Atrasti.API/Models/Profile/ProfilePage_Res.cs
using Atrasti.API.Models.User;

namespace Atrasti.API.Models.Profile
{
    public class ProfilePage_Res
    {
        public bool Setup { get; set; }
        public bool IsProfileOwner { get; set; }
        public User_Res User { get; set; }
        public CompanyPage_Res CompanyPage { get; set; }
        public AgentPage_Res AgentPage { get; set; }
        public int UserType { get; set; }
    }
}